iron-flex-layout.html

```diff
@@ -259,6 +265,43 @@
       align-self: stretch;
     };
 
+    --layout-self-baseline: {
+      -ms-align-self: baseline;
+      /* currently not supported in safari */
+      -webkit-align-self: baseline;
+      align-self: baseline;
+    };
+
+    /* multi-line alignment in main axis */
+
+    /* currently not supported in ie */
+    /* currently not supported in safari */
```

Comment:
Oh interesting. I'm not sure how I feel about adding things that are so browser specific. We try to have Polymer elements work the same in all browsers, and I think this is misleading, even with a comment :(